### DISCUSSION TITLE
n:class as array

### DISCUSSION THREAD

**Discussion by emololftw** [nette/latte#293] [Ideas]
Dear community,

many times im using condition with n:class tag like:

```
n:class="$presenter->isModuleCurrent($item->module) ? text-white, $presenter->isModuleCurrent($item->module) ? bg-dark"
```

for more than one condition im using same rule, what about present each n:class condition as array key=>value, something like:

```
n:class="[ 
    $presenter->isModuleCurrent($item->module): 'text-white bg-dark and-another-one mt-5', 
    $conditionNext: 'item-separe mt-2'
]"
```

**Comment by dg** [comment]
This goes against PHP conventions because the values of keys must be different in arrays. Why doesn't this seem ok to you?

```
n:class="
    $presenter->isModuleCurrent($item->module) ? 'text-white bg-dark and-another-one mt-5', 
    $conditionNext ? 'item-separe mt-2'
"
```

**Reply by emololftw** [reply]
oohhhh latte supports multiple class definition in one condition, which must be included between quotes. I'm so sorry. This destroyed my post. Thank you for answer!
